src/housekeeping.rs

```diff
@@ -237,10 +245,21 @@ pub fn refresh_known_filters(
     Ok(0)
 }
 
+pub fn get_known_filters(
+) -> JoshResult<std::collections::BTreeMap<String, std::collections::BTreeSet<String>>> {
+    let mut kf = std::collections::BTreeMap::new();
+    for (k, v) in KNOWN_FILTERS.lock()?.iter() {
```

Comment:
can variables have more meaningful names here? or have a comment about what are k and v